src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -687,13 +685,13 @@ g.test('empty_bind_group_layouts_requires_empty_bind_groups,compute_pass')
     });
 
     const encoder = t.device.createCommandEncoder();
-    const pass = encoder.beginComputePass();
-    pass.setPipeline(pipeline);
+    const computePass = encoder.beginComputePass();
+    computePass.setPipeline(pipeline);
     for (let i = 0; i < bindGroupLayoutEntryCount; i++) {
-      pass.setBindGroup(i, emptyBindGroup);
+      computePass.setBindGroup(i, emptyBindGroup);
     }
-    pass.dispatchWorkgroups(0);
-    pass.end();
+    t.doCompute(computePass, 'dispatch', true);
```

Comment:
Should be parameterized over the 2 values in `kComputeCmds`

Reply:
Yes, right. I was also applying the parameterization to this test. I apply the parameterization to this PR as well.